docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -310,19 +311,62 @@ export const resolveTypeName = ({
 
 export const parseParamName = (name: string) => (name.startsWith('__') ? name.substr(2) : name);
 
-export const renderParam = ({ comment, name, type, flags }: MethodParamData): JSX.Element => (
-  <LI key={`param-${name}`}>
-    <B>
-      {parseParamName(name)}
-      {flags?.isOptional && '?'} (<InlineCode>{resolveTypeName(type)}</InlineCode>)
-    </B>
-    <CommentTextBlock comment={comment} components={mdInlineComponents} withDash />
-  </LI>
+export const renderParamRow = ({ comment, name, type, flags }: MethodParamData): JSX.Element => {
```

Comment:
Same comment as in the other API docs PR: we should convert these to be React components instead of plain functions. By doing that we get a few benefits:
- More helpful information in the react profiler (since we get component names)
- We also make the markup part of the component nearly all markup
- Components are also the pattern we use on the website (instead of functions)

Reply:
If I do not make the refactor PR, I'm trying to stick to the locally applied patterns, so I have decide to implement those as helpers, like other parts of the `APISection` components.

Generally, at the beginning of coding this feature I have tried to avid defining more components then necessary, because first implementation included too much of them, and IIRC this approach is a legacy of the past. Fully agree that having those helpers as components could add the benfits.